src/components/login/login-button.tsx

```diff
@@ -2,18 +2,19 @@
 
 import { signIn } from 'next-auth/react';
 import {
-    Button,
     ButtonSize,
-    IconProfile,
+    IconMumble,
+    NaviButton,
 } from '@ost-cas-fee-adv-23-24/design-system-pixelpioneers';
 
 export default function LoginButton() {
     return (
-        <Button
```

Comment:
We can union the login and logout button (and call it just LoginButton) to one component with a boolean prop like loggedIn? = false to differentiate if the user is logged in or not